CSCI504-Assignment4/Form1.cs

```diff
@@ -17,11 +17,12 @@ public partial class Form1 : Form
         private Pen pen = new Pen(Color.Black, 1);
         private Point start;
         private Point finish;
-        private Stack<Line> undoLines = new Stack<Line>();
+        private List<Line> undoLines = new List<Line>();
```

Comment:
The lines collection needs to be a stack in order to support the undo and redo functionality.